zebra/zebra_vrf.c

```diff
@@ -216,8 +220,21 @@ static int zebra_vrf_disable(struct vrf *vrf)
 		 * we no-longer need this pointer.
 		 */
 		for (safi = SAFI_UNICAST; safi <= SAFI_MULTICAST; safi++) {
-			zebra_router_release_table(zvrf, zvrf->table_id, afi,
-						   safi);
```

Comment:
I'm unclear about removing this: if there's an empty table, wouldn't it be ok to call 'release' on it?

Reply:
I now only remove it it is empty